src/pykx/pandas_api/pandas_meta.py

```diff
@@ -228,6 +228,14 @@ def max(self, axis=0, skipna=True, numeric_only=False):
             res
         ), cols)
 
+    @convert_result
+    def idxmax(self, axis=0, skipna=True, numeric_only=False):
+        tab = self
+        res, cols = preparse_computations(tab, axis, skipna, numeric_only)
+        col_names = _get_numeric_only_subtable_with_bools(tab)[1] if numeric_only else tab.columns
+        max_vals = [elems.index(max(elems)) for elems in res]
```

Comment:
I believe this is broken, `elems` has forgotten original indexes. Take a look at this case:
```python
>>> t
pykx.Table(pykx.q('
a b
---
  1
1 2
2 3
'))
>>> t.pd()
     a  b
0  NaN  1
1  1.0  2
2  2.0  3
>>> t.pd().idxmax()
a    2
b    2
dtype: int64
>>> t.idxmax()
pykx.Dictionary(pykx.q('
a| 1
b| 2
'))
```